changelog/undistributed.rst

```diff
@@ -13,3 +13,5 @@
 --------------------------------------------------------------------------------
                                 Fix
 --------------------------------------------------------------------------------
+* Update ShowIpRoute:
```

Comment:
Hi @leoanez ,

Thanks for the update, would you mind add the os, `* IOSXE`, above it?

Thanks,
Irene

Reply:
Done